requirements/base.txt

```diff
@@ -7,16 +7,17 @@ kornia==0.5.6
 lxml==4.6.5
 matplotlib==3.4.3
 networkx~=2.5
-numpy~=1.19.5
+# nncf==2.1.0
+numpy>=1.22.3
 omegaconf==2.1.1
+pandas==1.4.1
 opencv-python>=4.5.3.56
 opencv-contrib-python==4.5.5.62
-pandas~=1.1.5
 pillow==9.0.0
-pytorch-lightning==1.5.9
-torch==1.8.1
-torchvision==0.9.1
-torchtext==0.9.1
+pytorch-lightning==1.6.0
```

Comment:
pytorch-lightning 1.6.0 modified some of the annotations under the hood. CI is failing due to these changes. 
[CI-PR-158.log](https://github.com/openvinotoolkit/anomalib/files/8427463/CI-PR-158.log)

Reply:
This is a problem because I cannot install torchtext 0.9.1 but it does not seem that pytorch lightning 1.5.9 works with 0.12.0.

Reply:
can you try the below requirements file to see if it works on you mac?
https://github.com/openvinotoolkit/anomalib/blob/fix/sa/requirements/requirements/base.txt

Reply:
With those requirements, anomalib tests pass. They shoudl be working fine with your mac too.

Reply:
kornia needs to be tied down to 0.5.6 otherwise it throws this error ImportError: cannot import name 'gaussian_blur2d' from 'kornia'

matplotlib seems to be missing from the requirements somehow. I changed my base to match yours and added matplotlib and fixed the kornia dependency.

Reply:
regarding the kornia, looks like we need to modify the import as follows:

~~from kornia import gaussian_blur2d~~
from kornia.filters import gaussian_blur2d
 in `anomalib/models/patchcore/model.py` and `anomalib/models/padim/model.py`

Reply:
Yep that's right. I also checked that import with the 0.5.6 and it seems to work too. Pushed the change. I can't seem to get the test suite to autodiscover on my machine as it's throwing errors in albumentations but I can give it a try tomorrow. Are tests passing on yours?

Reply:
When I use the base requirements that I shared with you, tests pass. I used #193 as well, though. I'll be merging it, so you might want to pull the latest develop branch after I merge it. I could trigger your tests afterwards.

Reply:
Just merged PR #193 now.

Reply:
Merged here.

Reply:
Thanks, I'll run the CI now.

Reply:
CI failed due to sorting imports. Can you run 

```
isort  --df anomalib --profile=black
```

from your terminal and push the changes?

Reply:
Ah sorry, left an extra line. Done!

Reply:
@samet-akcay, saw the CI failed. Pushed a fix for tox. Would you mind running the workflow again please?

Reply:
@dreaquil, as I mentioned above, if you could set `wandb==0.12.9`, I could re-run the CI, which would hopefully pass this time.

Reply:
Haha, okay... this should be it! @samet-akcay